extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php

```diff
@@ -115,6 +115,7 @@
 	'user-identity-remove-fail' => 'There was some problem when removing the avatar!',
 	'user-identity-avatars-maintenance' => 'Upload and deletion of avatars temporarily disabled during maintenance!',
 	'userprofilepage-edit-modal-error' => 'Unexpected error occurred',
+	'user-identity-token-mismatch' => 'Oops! Token doesn\'t match',
```

Comment:
The standard message MediaWiki shows when the provided token is incorrect is [`sessionfailure`](https://github.com/Wikia/app/blob/1dfb4dec20c0/languages/messages/MessagesEn.php#L2886-L2888), so that or a variant on it is an option.